cabal.project

```diff
@@ -4,3 +4,9 @@ constraints: ormolu +dev
 
 -- miso does not specify bounds on servant
 constraints: servant >=0.18 && <0.20
+
+-- prune unnecessary dependencies
+constraints: warp -x509, wai-app-static -cryptonite
```

Comment:
Not on CI, but when you locally enter the `ormolu-live` directory, this will mean that you don't have to compile the big transitive dependency tree of `tls` (`basement`, `cryptonite` etc.).